### ISSUE TITLE
Cannot find Torrent2HTTP

### ISSUE THREAD

**Issue by JanV123** [markop159/KODI-Popcorn-Time#102]
Hi,

I recently upgraded from Kodi 17.6 to 18.0. When I was still on 17.6 Kodi Popcorn Time was working fine but when I uninstalled 17.6 and moved to 18.0, I am getting a "Cannot Find Torrent2HTTP" error when trying to open Popcorn Time videos. Hoping for any assistance.
[Here](https://paste.kodi.tv/yifojofepi) are my logs.

**Comment by markop159** [comment]
Hi,

New version of Kodi on Windows is running as x64 application, so it is searching for x64 binnaries of torrent2http. To fix this you need to go this location on your PC:

```
"C:\Users\Admin\AppData\Roaming\Kodi\addons\plugin.video.kodipopcorntime\resources\bin"

rename folder "windows_x86" to "windows_x64"
```

I will update plugin with a fix when I have some time.

**Comment by JanV123** [comment]
> Hi,
> 
> New version of Kodi on Windows is running as x64 application, so it is searching for x64 binnaries of torrent2http. To fix this you need to go this location on your PC:
> 
> ```
> "C:\Users\Admin\AppData\Roaming\Kodi\addons\plugin.video.kodipopcorntime\resources\bin"
> 
> rename folder "windows_x86" to "windows_x64"
> ```
> I will update plugin with a fix when I have some time.

Thanks buddy. Confirmed to be all working now!

Cheers!

**Comment by Jonyx4** [comment]
Should you add this _issue_ to the installation instructions page?

**Comment by no1be4me** [comment]
Is there a fix for 1.75 on Kodi 18.6 on Raspberry Pi?
